vendor/engines/ldap_authentication/ldap_authentication.gemspec

```diff
@@ -16,6 +16,6 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "README.md"]
   s.test_files = Dir["spec/**/*"]
 
-  s.add_dependency "rails", "~> 4.2.7.1"
+  s.add_dependency "rails", "~> 4.2.9"
```

Comment:
Was there a specific reason we included the rails dependency on this engine? Most of the other engines don't include rails as a dependency, even though they use it...